packages/graphback-datasync/tests/DataSyncMongoDBTests.ts

```diff
@@ -108,4 +108,39 @@ describe('Soft deletion test', () => {
     // Tests that we cannot find it anymore
     expect(Post.findOne({id})).rejects.toThrow();
   })
+
+  it('can filter sync queries', async () => {
+    context = await createTestingContext(`
+    """
+    @model
+    @versioned
+    @delta
+    """
+    type Post {
+      id: ID!
+      text: String!
+    }
+    `);
+
+    const { Post }  = context.providers;
+
+    // Create some posts
+    for (const postTitle of ["post", "post2", "trains"]) {
+      await Post.create({ text: postTitle });
+    }
+
+    // Sync query
+    const deltaPosts = await Post.sync("0", {
```

Comment:
I would recommend to do second sync with return timestamp and check if result is zero etc. (maybe separate tests)

Reply:
On it